src/physics/utils/orbital_data.F90

```diff
@@ -0,0 +1,78 @@
+! Copyright (C) 2024 National Science Foundation-National Center for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+module orbital_data
+!--------------------------------------------------------------------------
+!
+! Provides access to conditions calculated based on the Earth's orbit.
+!
+!--------------------------------------------------------------------------
+
+  use shr_kind_mod,  only: kind_phys => SHR_KIND_R8
+  use shr_const_mod, only: PI => SHR_CONST_PI
```

Comment:
Also forgot to add that I don't think (?) `PI` is used anywhere in this module.

Reply:
removed!